plugins/source/slack/resources/services/conversations/conversation_bookmarks_fetch.go

```diff
@@ -0,0 +1,27 @@
+package conversations
+
+import (
+	"context"
+
+	"github.com/cloudquery/cloudquery/plugins/source/slack/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchConversationBookmarks(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	c := meta.(*client.Client)
+	f := func() error {
+		channelID := parent.Get("id").String()
```

Comment:
nice, will need to remember it

Reply:
I need to double-check this is right actually 😅

Reply:
I think it's fine to do this as long as we don't plan on changing the behavior of the CQType `String()` methods. Currently they return the contents verbatim